src/quo/components/list/item.cljs

```diff
@@ -198,7 +198,7 @@
            title subtitle subtitle-secondary active on-press on-long-press chevron size text-size
            accessory-text accessibility-label title-accessibility-label accessory-style
            haptic-feedback haptic-type error animated animated-accessory? title-text-weight]
-    :or   {subtitle-max-lines 1
+    :or   {subtitle-max-lines 0
```

Comment:
I'm not sure we want to change the default value for the component, better to pass a parameter for this case only

Reply:
Hi @flexsurfer ,
I restored the default value and passed the parameter to 4 instances where overflowing was happening.